spec/forms/tell_us_about_the_new_job_form_spec.rb

```diff
@@ -35,22 +31,20 @@
 
     describe "manager_name" do
       context "when the manager_name is not included" do
-        it "is invalid" do
-          invalid_params = valid_params.merge(manager_name: nil)
-          form = TellUsAboutTheNewJobForm.new(nil, invalid_params)
+        it "is valid" do
+          form = TellUsAboutTheNewJobForm.new(nil, valid_params.merge(manager_name: nil))
 
-          expect(form).to_not be_valid
+          expect(form).to be_valid
         end
       end
     end
 
     describe "manager_phone_number" do
       context "when the manager_phone_number is not included" do
-        it "is invalid" do
-          invalid_params = valid_params.merge(manager_phone_number: nil)
-          form = TellUsAboutTheNewJobForm.new(nil, invalid_params)
+        it "is valid" do
+          form = TellUsAboutTheNewJobForm.new(nil, valid_params.merge(manager_phone_number: nil))
 
-          expect(form).to_not be_valid
+          expect(form).to be_valid
         end
       end
 
```

Comment:
can we add a test in here for submitting an empty string as the manager phone number, since that's the behavior we're seeing? 

here's an example I did from the similar branch:
```
      context "when the manager_phone_number is a blank string" do
        it "is valid" do
          params = valid_params.merge(manager_phone_number: "")
          form = TellUsAboutTheLostJobForm.new(nil, params)
          expect(form).to be_valid
        end
      end
```

Reply:
Good point, used your example.